spec/models/generic_file_spec.rb

```diff
@@ -214,6 +214,103 @@
     end
   end
 
+  describe 'callbacks for preservation' do
+    let(:generic_file) do
+      FactoryGirl.build(:generic_file, title: ['Test Title'], creator: ['John Doe'], resource_type: ['Book']) do |gf|
+                          gf.visibility = Hydra::AccessControls::AccessRight::VISIBILITY_TEXT_VALUE_PUBLIC
+                          gf.apply_depositor_metadata('ditest@example.com')
+                        end
+    end
+
+    before(:each) do
+      # clear out the test preservation queue for consistent results
+      $redis.del Hydranorth::PreservationQueue::QUEUE_NAME
```

Comment:
$redis goes through the raw redis connection not set up in PreservationQueue. Testing through a separate connection is just my attempt to make the test logic not depend on the preservation logic in any way